### PR TITLE
EZP-28495: As Editor I want UI for external links management

### PR DESCRIPTION
**Pull request by adamwojs**
> JIRA: https://jira.ez.no/browse/EZP-28495
> **Requires: https://github.com/ezsystems/ezpublish-kernel/pull/2183**

## Description

This PR contains common components for the link management UI for v1 and v2, including:

* URL list/update form definition
* DTOs for URL list/update form
* Pagerfanta adapters for listings URLs and content items using URL  
* URL unique constraint

### CODE REVIEW

**Comment by lserwatka** [comment]
@adamwojs we have to make it other way around. First we need to make a new branch 1.11 (with branch alias 1.11.x-dev) out of 1.10. Then change target branch to 1.11 here. Then once merged it should be merge up to master (2.0.x-dev).

**Comment by lserwatka** [comment]
We are aiming for 1.x with this feature thus 1.13 must have dependency on repository-forms 1.11

**Comment by adamwojs** [comment]
> @adamwojs we have to make it other way around. First we need to make a new branch 1.11 (with branch alias 1.11.x-dev) out of 1.10. Then change target branch to 1.11 here. Then once merged it should be merge up to master (2.0.x-dev).

@lserwatka Done.

File: lib/Data/URL/URLListData.php [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
+ * This file is part of the eZ RepositoryForms package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
```
Comment (alongosz) [inline lib/Data/URL/URLListData.php]: Nitpick: skip in all files.

Comment (andrerom) [reply lib/Data/URL/URLListData.php]: I'll fix that

**Pull request merged**